docs_src/microservices/application/Configuration.md

```diff
@@ -0,0 +1,74 @@
+# Application Services Configuration
```

Comment:
I dont think this title works -- it implies that it is the configuration for any service that uses the SDK which isn't the case. I think the best thing to do is simply call this section "Configuration" and nest it under App Service Configurable.

Reply:
This is for any Application Service.

Reply:
Changed to `General App Service Configuration` so it is clearer.